src/app/api/common/votes/getVotes.ts

```diff
@@ -231,12 +230,12 @@ async function getVotesForProposalAndDelegate({
   proposal_id: string;
   address: string;
 }) {
-  const { namespace } = Tenant.current();
+  const { namespace, contracts } = Tenant.current();
   const votes = await prisma[`${namespace}Votes`].findMany({
```

Comment:
not related to your PR but wanted to flag that while attempting to setup the scroll tenant I kept running into needing these `prisma[${namespace}-x]` views in the prisma schema. It's pretty tedious to do this per tenant and I hope it actually scales okay (meaning do all tenants have the same views, or is this governor based) 

something to flag for monday